scripts/lib/k8s.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+CONTROLLER_TOOLS_VERSION="v0.4.0"
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$THIS_DIR/../.."
+SCRIPTS_DIR="$ROOT_DIR/scripts"
+
+# controller_gen_version_equals accepts a string version and returns 0 if the
+# installed version of controller-gen matches the supplied version, otherwise
+# returns 1
+#
+# Usage:
+#
+#   if controller_gen_version_equals "v0.4.0"; then
+#       echo "controller-gen is at version 0.4.0"
+#   fi
+k8s_controller_gen_version_equals() {
```

Comment:
This is unused I think?

Reply:
Yeah I was thinking of checking within the `build-runtime.sh` script, again. Since there is a chance someone is calling the script directly, rather than through any of the scripts in `community`. But I don't like the code duplication.